docs/release/Betaflight-4.4-Release-Notes.md

```diff
@@ -1,10 +1,10 @@
 # Betaflight 4.4 Release Notes
 
-## 1. Cloud build
+## 1. Cloud Build
 
 This is predominantly brought to you for convenience, and to ensure we can keep the 512kb flash targets (STM32F411 and STM32F722) alive and well for years to come. The cloud build system will allow you, the flyer, to select the features you want, and a **custom firmware** will be created for you.
 
-Any issues with the cloud build process please check out the #cloud-build-issues Discord channel. Help us to help you, buy taking advantage of the new `Support` button in the Command Line Interface (CLI) tab in Configurator 10.9.0 (RC4 onwards). This will give us valuable information in trying to diagnose your issue.
+For any issues with the cloud build process please check out the #cloud-build-issues channel on the [Discord server](https://discord.betaflight.com/invite). Help us to help you, buy taking advantage of the new `Support` button in the Command Line Interface (CLI) tab in Configurator 10.9.0 (RC4 onwards). This will give us valuable information in trying to diagnose your issue.
```

Comment:
```suggestion
For any issues with the cloud build process please check out the #cloud-build-issues channel on the [Discord server](https://discord.betaflight.com/invite). Help us to help you, buy taking advantage of the new `Support` button in the Command Line Interface (CLI) tab in Configurator 10.9.0. This will give us valuable information in trying to diagnose your issue.
```